yellowbox_heksher/heksher_service.py

```diff
@@ -14,24 +15,33 @@
 
 class HeksherService(SingleEndpointService, RunMixin):
     def __init__(self, docker_client: DockerClient, postgres_image: str = 'postgres:latest',
-                 heksher_image: str = 'biocatchltd/heksher:0.3.0', port: int = 0, *,
+                 heksher_image: str = 'biocatchltd/heksher:0.4.1', port: int = 0, *,
                  heksher_startup_context_features: str, **kwargs):
         creator = SafeContainerCreator(docker_client)
 
         self.postgres_service = PostgreSQLService(docker_client, image=postgres_image, default_db='heksher')
 
+        self.heksher_db_migrator = creator.create_and_pull(
+            heksher_image, "alembic upgrade head", publish_all_ports=True, ports={80: port}, detach=True, environment={
+                'HEKSHER_DB_CONNECTION_STRING': self.postgres_service.container_connection_string("postgres"),
+                'HEKSHER_STARTUP_CONTEXT_FEATURES': heksher_startup_context_features,
+            }
+        )
```

Comment:
a. not sure why this is an attribute, don't we only need it during startup?
b: also I don't think alembic exposes any ports.
c: startup context features are also unnecessary